testsuite/integration/run.sh

```diff
@@ -3,16 +3,18 @@ set -ueo pipefail
 set +x
 
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd $SCRIPTDIR
 
-cd $SCRIPTDIR && go install \
-	storj.io/storj/cmd/certificates \
-	storj.io/storj/cmd/identity \
-	storj.io/storj/cmd/satellite \
-	storj.io/storj/cmd/storagenode \
-	storj.io/storj/cmd/versioncontrol \
-	storj.io/storj/cmd/storj-sim
+VERSION=$(go list -m -f "{{.Version}}" storj.io/storj)
 
-cd $SCRIPTDIR && go install storj.io/gateway
+go install storj.io/storj/cmd/certificates@$VERSION
```

Comment:
iirc `go install ... @version` doesn't work in our images because of the `cc` magic script which will turn this into a `go build`. In the other place where this was desired I used `/usr/local/bin/go` to circumvent the magic.

Reply:
fixed to use the specific binary